net/uring/netenc/big.go

```diff
@@ -0,0 +1,14 @@
+// +build mips mips64 ppc64 s390x
+
+package netenc
+
+// TODO how should I capitalize this
```

Comment:
Good question. The standard library (strong) uses Atoi, which suggests Ntoh. We could also do NetworkToHost, which is nice and explicit. I think either would be OK. On a second look, we should probably use Ntoh16 instead of Ntohs, though, following math/bits and encoding/binary naming conventions.